docs/Guide/CLI/generating-components.md

```diff
@@ -0,0 +1,26 @@
+---
+title: Generating components
+sidebar_position: 2
+---
+
+We can generate components using:
+
+```bash
+sapphire generate <component> <name>
+```
+
+- **component**: Component name, can be a default one or a custom one
+- **name**: Name of the created component, e.g. the name of your command
+
+Example:
+
+```bash
+sapphire generate command HelloWorld
+```
+
+## Default components
```

Comment:
```suggestion
## Default categories
```

Reply:
@enxg categories vs components, what do you say

Reply:
> @enxg categories vs components, what do you say

Explained this on discord